### ISSUE TITLE
docs on read.adp should be more explicit on value (e.g. nav)

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#703]
As I was coding a solution for #696, I realized that the docs should list the details of the value more precisely. For example, it should say that ``slongitude`` means such-and-such, and that it is named so-and-so on page blab-blah of the document name who-who etc. That way, users are encouraged to go to the RDI manual, which is where they should be looking to see what their instrument is doing.

**Comment by dankelley** [comment]
Done in ``develop``. Note that I removed the details from the ``read.adp()`` docs; they belong in the ``adp-class`` docs. What I've written is quite brief but probably sufficient to close this.

**Comment by richardsc** [comment]
Great. I'll have a look, and possibly do a PR request if I think anything should be added/edited.

**Comment by richardsc** [comment]
I'm just having a look at the changes now. I'm going to put my responses to your "in code" comments from the `adp-class` man page here. I'll let you edit your comments as you make any changes.

1. Re: `slatitude` and `elatitude` etc names. In principle I actually think I would prefer `startLatitude` and `endLatitude`, since it's more explicit. There is the problem of breaking old things, but I don't think too many people have been using the VMDAS capabilities yet (except for me and @gbennett02) so it shouldn't be catastrophic. The only reason I called them `slatitude` etc is because those are the names Rich P uses in is Matlab `rdradcp.m` code.

2. From the `?'adp-class'` help page FIXME 1: I can't tell what `avgTrackTrue` really means relative to the GUI. I'll try poking around in it some more, but my guess is that it is "True".

3. From the `?'adp-class'` help page FIXME 2: `navTime` or "ADCP Ensemble time" is displayed in the GUI as the title in the "Data Quality" box. Note that the `navTime` field should be exactly the same as the regular "ADCP time", i.e. `d[['time']]`. Using `testthat` to check should work -- it did for me, though it doesn't like that the order of the classes are different:
```
> expect_equal(d[['navTime']], d[['time']])
Error: d[["navTime"]] not equal to d[["time"]]
Attributes: < Component “class”: 2 string mismatches >
target is POSIXt, current is POSIXct
> d[['navTime']][1]
[1] "2012-11-14 09:20:10.17 UTC"
> d[['time']][1]
[1] "2012-11-14 09:20:10.17 UTC"
> class(d[['navTime']][1])
[1] "POSIXct" "POSIXt" 
> class(d[['time']][1])
[1] "POSIXt"  "POSIXct"
```

4. From the `?'adp-class'` help page FIXME 3: Given that the `navTime` field is redundant, I don't think there's a high priority to rename it. I would consider renaming all the `s*`/`e*` things to be `start*`/`end*`, though, depending on what you think.

**Comment by dankelley** [comment]
1. A trick: if you start the code block in your item 3 with a space before the double-backtick, then the list items continue to flow. I edited your comment to do that.

2. I'll change the names to e.g. ``start*``.

3. This is probably good to have in mind, and I'll figure out why but ... see next item.

4. I plan to simply remove the ``navTime``. It does seem to be a duplicate of a time we already obtain. (The docs suggest the same thing.)

**Comment by dankelley** [comment]
I updated code, doc and oce-issues test. The test is clean. (I did most of this in Rstudio.  It is quite nice  for quick build/test cycles. However, I really don't like the all-windows-in-one approach. I have a window system for that, and it lets me overlap, etc.)

Since I don't think we can infer whether the gui "Speed/Avg/Dir" is "true" or "magnetic", I put question marks in the table. Probably we'll find that out someday, when a user who has nonzero values asks us, and we run the code and see.

Since you reopened this, may I ask you to make the decision on whether to close it?
